Snakefile

```diff
@@ -497,8 +497,9 @@ onsuccess:
         shell("./bin/clean")
 
 onerror:
```

Comment:
Noticed while reviewiing this PR, and perhaps out-of-scope. In the `onsuccess` block immediately above this, the message will not be sent to slack, correct? Could we add a `"$bin"/notify-slack "{message}"` shell block there? (It would be better if it were threaded, but I remember this not being straightforward with the ncov-ingest set-up.)

Reply:
Hmm, I thought an `onsuccess` message would just be extra noise. There's already slack notifications for the file updates, which indicate the pipeline successfully completed. The successful pipeline will also trigger the ncov builds, which will send the "Pipeline starting..." messages.

Reply:
Good points Jover, I agree!